OpenRA.Game/StreamExts.cs

```diff
@@ -133,15 +186,14 @@ public static byte[] ReadAllBytes(this Stream s)
 				int count;
 				while ((count = s.Read(buffer, 0, buffer.Length)) > 0)
 					bytes.AddRange(buffer.Take(count));
+
 				return bytes.ToArray();
 			}
 		}
 
-		// Note: renamed from Write() to avoid being aliased by
-		// System.IO.Stream.Write(System.ReadOnlySpan) (which is not implemented in Mono)
```

Comment:
If this is solved then the method should be removed and callers should directly call `Write`?

Reply:
Sure, removed the helper.